easybuild/easyconfigs/j/jax/jax-0.2.18-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'jax'
+version = '0.2.18'
+
+homepage = 'https://pypi.python.org/pypi/jax'
+description = """Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and
+more"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/google/jax/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+patches = [
+    'jax-0.2.18-correct_libraries.patch',
+]
+
+preinstallopts = 'python3 build/build.py '
+preinstallopts += '--bazel_startup_options="--output_user_root=%(builddir)s" '
+preinstallopts += ' --bazel_path="$EBROOTBAZEL/bin/bazel" && '
+preinstallopts += 'pip install dist/*.whl && '
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Bazel', '3.7.2'),
+    ('SciPy-bundle', '2020.11'),
+    ('Cython', '0.29.22'),
```

Comment:
Can you fix the deps please?
- Bazel is a build dep
- Is Cython really required?
- This whole thing needs to be a PythonBundle as absl-py and opt-einsum are required
- flatbuffers-python is also missing

Reply:
done